nvflare/apis/utils/fl_context_utils.py

```diff
@@ -89,3 +90,14 @@ def generate_log_message(fl_ctx: FLContext, msg: str):
             ctx_items.append(item + "=" + str(all_kvs[item]))
 
     return "[" + ", ".join(ctx_items) + "]: " + msg
+
+
+def add_job_audit_event(fl_ctx: FLContext, ref: str = "", msg: str = "") -> str:
+    return AuditService.add_job_event(
+        job_id=fl_ctx.get_job_id(),
+        scope_name=fl_ctx.get_prop(FLContextKey.EFFECTIVE_JOB_SCOPE_NAME, "?"),
+        task_name=fl_ctx.get_prop(FLContextKey.TASK_NAME, "?"),
+        task_id=fl_ctx.get_prop(FLContextKey.TASK_ID, "?"),
+        ref=ref,
+        msg=msg,
+    )
```

Comment:
should this function in here or in `AuditService`?

Reply:
This is a convenience function for people to use. Do not belong to AuditService.